config/prod.exs

```diff
@@ -51,4 +51,4 @@ config :logger, level: :info
 
 # Finally import the config/prod.secret.exs which loads secrets
 # and configuration from environment variables.
-import_config "prod.secret.exs"
+# import_config "prod.secret.exs"
```

Comment:
Have you tried running the local phoenix server in prod env, i.e. `MIX_ENV=prod mix phx.server`.

This failed when I tried executing this locally because database credentials were missing. We should enable this.